index.html

```diff
@@ -1,9 +1,13 @@
 <!DOCTYPE html>
 <html>
-<head>
-<title>Okta Documentation</title>
```

Comment:
Is the title removed on purpose? It looked like a nice addition

Reply:
I guess it can't hurt, added it back. thanks!